Command was updated due to API changes in the new version (22.06) (#20833)
demisto/content: Packs/PrismaCloudCompute/ReleaseNotes/1_2_16.md

#### Integrations
##### Palo Alto Networks - Prisma Cloud Compute

> - ***Breaking Change*** Fixed an issue in **prisma-cloud-compute-vulnerabilities-impacted-resources-list** command following changes in the new released version.

Reviewer: if there are context to argent changes that are breaking, we should consider reporting them in the release notes. let's talk about that.

#### Integrations
##### Palo Alto Networks - Prisma Cloud Compute

> - ***Breaking Change*** As a result of a change in the new version (22.06) of Prisma Cloud Compute, the **prisma-cloud-compute-vulnerabilities-impacted-resources-list** command has been updated:
>   - Added support for an argument called *resourceType*, which can be one of the following: 'container','image','host','function','codeRepo','registryImage'.
>   - Note: Due to product's limitation, when resourceType isn't sent - images and hosts returns by default.
>   - Added a *resourceID* to the response. 

Reviewer 2: ```suggestion
- ***Breaking Change*** As a result of a change in the new version (22.06) of Prisma Cloud Compute, the ***prisma-cloud-compute-vulnerabilities-impacted-resources-list*** command has been updated to:
  - Added support for the argument *resourceType*, which can be one of the following options: 'container', 'image', 'host', 'function', 'codeRepo', or 'registryImage'.
  - Note: Due to a product limitation, when resourceType isn't sent, images and hosts will return by default.
  - Added a *resourceID* to the response. 
```
